doc/man/man3/crypto_blake2b.3monocypher

```diff
@@ -104,6 +104,11 @@
 BLAKE2b is a fast cryptographically secure hash, based on the ideas of
 Chacha20.
 It is faster than MD5, yet just as secure as SHA-3.
+However, BLAKE2b itself
+.Sy is not suitable for hashing and deriving keys from passwords ;
```

Comment:
This sounds like Blake2b is not suitable for general hashing while you actually mean "not suitable for hashing passwords and deriving key from passwords"

Reply:
Point given. I was trying to be brief, but that was probably abbreviating a bit much.